### PR TITLE
Bump timeout from 1 to 10 seconds

### PR DESCRIPTION
**Pull request by hardillb**
Mitigation for #17

### CODE REVIEW

**Comment by knolleary** [comment]
Why not make it 5 or 10 seconds? 2 seconds is still longer than we want it to take, but it doesn't really give any head room over the existing 1 second timeout.

**Comment by hardillb** [comment]
True, a 100% increase just felt like a lot.

**Comment by sammachin** [comment]
Agree with Nick, the timeout should only be triggered in an error scenario so for the 99.9% of requests that already happen in <1sec there's no difference but the few that fail now would be better to be slow than not at all, we can then continue to monitor this and see if we still see errors at 10sec because something is actually failing rather than slow or we can look to bring it back down to 2-3s later

**Comment by hardillb** [comment]
OK, will make 10 seconds.

File: index.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ module.exports = {
                 authorization: 'Bearer ' + settings.token
             },
             timeout: {
-                request: 1000
+                request: 2000
```
Comment (hardillb) [inline index.js:27]:
```suggestion
                request: 10000
```

**Review by sammachin — APPROVED** [review]

**Pull request merged**